cloud_governance/common/clouds/aws/utils/utils.py

```diff
@@ -34,3 +36,20 @@ def get_details_resource_list(self, func_name: Callable, input_tag: str, check_t
                 resources = func_name(Marker=resources[check_tag], **kwargs)
             resource_list.extend(resources[input_tag])
         return resource_list
+
+    @typeguard.typechecked
+    @logger_time_stamp
+    def tag_aws_resources(self, client_method: Callable, tags: list, resource_ids: list):
+        """
+        This method tag the aws resources with batch wise of 20
```

Comment:
You can limit it to the number of CPU :
```
import multiprocessing
multiprocessing.cpu_count()
```
Also you can make and environment variable from TAG_BATCHES

Do you have 20 core on slave machine ?

Reply:
NO, we need to limit the CPU count.

Reply:
Its virtual CPU not physical CPU